localisation/000_Other_title_names.csv

```diff
@@ -1,349 +1,351 @@
+k_dun_niffelem;Dun Niffelem;Dun Niffelem;Dun Niffelem;;Dun Niffelem;;;;;;;;;x
+k_dun_niffelem_adj;Dun Niffeliam;;;;;;;;;;;;;x
+e_kolkar;Kolkar;Kolkar;;;Kolkar;;;;;;;;;x
 e_kolkar_adj;Kolkar;Kolkar;;;;;;;;;;;;x
-d_woodpaw;Woodpaw;;Waldpfoten;;;;;;;;;;;x
-d_woodpaw_adj;Woodpaw;;waldpfotisch;;;;;;;;;;;x
-d_savage_hill;Savage Hill;;;;;;;;;;;;;x
-d_savage_hill_adj;Savage Hill;;;;;;;;;;;;;x
-e_green_dragonflight;Green Dragonflight;Vol vert;Gr�ner Drachenschwarm;;;;;;;;;;;x
-e_green_dragonflight_adj;Verdant;Verdante;smaragdgr�n;;;;;;;;;;;x
-e_black_dragonflight;Black Dragonflight;Vol noir;Schwarzer Drachenschwarm;;;;;;;;;;;x
-e_black_dragonflight_adj;Obsidian;Obsidienne;pechschwarz;;;;;;;;;;;x
-e_red_dragonflight;Red Dragonflight;Vol rouge;Roter Drachenschwarm;;;;;;;;;;;x
-e_red_dragonflight_adj;Vermilion;Vermillon;zinnober;;;;;;;;;;;x
-e_blue_dragonflight;Blue Dragonflight;Vol bleu;Blauer Drachenschwarm;;;;;;;;;;;x
-e_blue_dragonflight_adj;Azure;Azure;azur;;;;;;;;;;;x
-e_bronze_dragonflight;Bronze Dragonflight;Vol bronze;Bronzener Drachenschwarm;;;;;;;;;;;x
-e_bronze_dragonflight_adj;Bronze;Bronze;bronzen;;;;;;;;;;;x
-d_silverwing_sentinels;Silverwing Sentinels;Sentinelles d'Aile-argent;W�chter von Silberwind;;;;;;;;;;;x
-d_silverwing_sentinels_adj;Sentinel;Sentinelle;W�chter;;;;;;;;;;;x
-d_cenarion_circle;Cenarion Circle;Cercle c�narien;Zirkel des Cenarius;;;;;;;;;;;x
-d_cenarion_circle_adj;Cenarion;C�narienne;cenarisch;;;;;;;;;;;x
-d_wardens;Wardens;Les Gardiennes;W�chter;;;;;;;;;;;x
-d_wardens_adj;Warden;Gardienne;der W�chter;;;;;;;;;;;x
-k_wildhammer;Grim Batol;Grim Batol;;;;;;;;;;;;x
+d_woodpaw;Woodpaw;Griffebois;Waldpfoten;;Zarpaleña;;;;;;;;;x
+d_woodpaw_adj;Woodpaw;Griffebois;waldpfotisch;;;;;;;;;;;x
+d_savage_hill;Savage Hill;Colline Sauvage;;;Cerro Salvaje;;;;;;;;;x
+d_savage_hill_adj;Savage Hill;Colline Sauvage;;;;;;;;;;;;x
+e_green_dragonflight;Green Dragonflight;Vol draconique vert;Grüner Drachenschwarm;;Vuelo Verde;;;;;;;;;x
+e_green_dragonflight_adj;Verdant;verdoyante;smaragdgrün;;verdeante;;;;;;;;;x
+e_black_dragonflight;Black Dragonflight;Vol draconique noir;Schwarzer Drachenschwarm;;Vuelo Negro;;;;;;;;;x
+e_black_dragonflight_adj;Obsidian;obsidienne;pechschwarz;;obsidiana;;;;;;;;;x
+e_red_dragonflight;Red Dragonflight;Vol draconique rouge;Roter Drachenschwarm;;Vuelo Rojo;;;;;;;;;x
+e_red_dragonflight_adj;Vermillion;vermillon;zinnober;;bermellón;;;;;;;;;x
+e_blue_dragonflight;Blue Dragonflight;Vol draconique bleu;Blauer Drachenschwarm;;Vuelo Azul;;;;;;;;;x
+e_blue_dragonflight_adj;Azure;azur;azur;;azur;;;;;;;;;x
+e_bronze_dragonflight;Bronze Dragonflight;Vol draconique de bronze;Bronzener Drachenschwarm;;Vuelo Bronce;;;;;;;;;x
+e_bronze_dragonflight_adj;Bronze;bronze;bronzen;;bronce;;;;;;;;;x
+d_silverwing_sentinels;Silverwing Sentinels;Sentinelles d'Aile-argent;Schildwachen der Silverwing;;Centinelas Ala de Plata;;;;;;;;;x
+d_silverwing_sentinels_adj;Sentinel;Sentinelle;schildwache;;Centinela;;;;;;;;;x

```

Comment:
Well, this is how WoW handles them as adjectives: https://de.wowhead.com/npc=3694/schildwache-selarin

How would it look like IG now?

Reply:
WoW doesn't handle it at all, that's the problem. Like, if there's an outpost or a group of them, WoW would call it "outpost of the <Sentinels>" or "group of <Sentinels>". But there's no adjective in it... In german it would be "Außenposten der <Schildwachen>" at best, but ck2 doesn't handle it that way, but like <adjective>e Außenposten. That's why I'd prefer to either go with "schildwachisch" here, because then the automatically added "e" makes sense, or stay with english, whenever it refers to a group and not a "real" landbased duchy.

Reply:
Makes sense. I'll do the necessary changes.